src/types/simpleEarn.ts

```diff
@@ -0,0 +1,126 @@
+export interface SimpleEarnProductListParams {
+  asset?: string;
+  current?: number;
+  size?: number;
+  recvWindow?: number;
+  timestamp: number;
```

Comment:
Just saw these. So for private API calls, you don't need to worry about providing recvWindow nor timestamp in any of the request params. When the SDK builds the request and prepares the authentication part of it, it will:
- Make a timestamp (Date.now())
- Apply the recv window in the general REST client config
- Build a request signature using api credentials + timestamp
- Build a request using this signature, api key, timestamp (important to use the same timestamp!) and send it to the API.

Having this as a request param might cause interference or fail auth/sign completely (please test this after implementing these changes). So recvWindow & timestamp should be removed from any request params - they're already handled automatically by the SDK.